src/reuse/__init__.py

```diff
@@ -53,14 +53,17 @@
     re.compile(r"^\.gitkeep$"),
     re.compile(r"^\.hgtags$"),
     re.compile(r".*\.license$"),
-    # SPDX files from https://spdx.github.io/spdx-spec/conformance/#44-standard-data-format-requirements
-    re.compile(r".*\.spdx$"),
-    re.compile(r".*\.spdx.(rdf|json|xml|ya?ml)$"),
     # Workaround for https://github.com/fsfe/reuse-tool/issues/229
     re.compile(r"^CAL-1.0(-Combined-Work-Exception)?(\..+)?$"),
     re.compile(r"^SHL-2.1(\..+)?$"),
 ]
 
+_IGNORE_SPDX_PATTERNS = [
+    # SPDX files from https://spdx.github.io/spdx-spec/conformance/#44-standard-data-format-requirements
+    re.compile(r".*\.spdx$"),
+    re.compile(r".*\.spdx.(rdf|json|xml|ya?ml)$"),
+]
```

Comment:
I like this idea a lot. But I think we can reduce the likelihood of errors like this:

```python
_IGNORE_SPDX_PATTERN = re.compile(r"combined pattern goes here")

_IGNORE_FILE_PATTERNS = [
    re.compile(r"^LICENSE"),
    [...],
    _IGNORE_SPDX_PATTERN,
]
```

This way you don't have to specifically check against `_IGNORE_FILE_PATTERNS + _IGNORE_SPDX_PATTERNS` elsewhere.

Reply:
@mxmehl @carmenbianca How can we move this along?  I think either version is fine.

Reply:
The proposal in @mxmehl 's last comment LGTM.